### ISSUE TITLE
Normaler RSS-Feed einlesen

### ISSUE THREAD

**Issue by zonky2** [pdir/social-feed-bundle#65]
Normaler RSS-Feed einlesen

**Comment by MDevster** [comment]
@zonky2 [contao-rssimport-bundle](https://extensions.contao.org/?q=rss%20import&pages=1&p=fipps%2Fcontao-rssimport-bundle) kennst du?

**Comment by zonky2** [comment]
ja, aber da eins ggf. für Twitter rein kommt, wäre "all-in-one" prima ;-)

von der grundlegenden Technik dürfte das ja soweit vorhenden sein

**Comment by MDevster** [comment]
@dev use https://github.com/dg/rss-php to import rss/atom feeds
